.gitignore

```diff
@@ -4,14 +4,15 @@
 /node_modules
 /.pnp
 .pnp.js
-
+package-lock.json
```

Comment:
The `package-lock.json` shouldn't be ignored _ever_, because it contains metadata about the installed packages that help `npm` to install the same set of dependencies across multiple different arquitectures.